assets/content/static/navbar.html

```diff
@@ -70,7 +70,7 @@ <h6 class="text-primary mb-md-1">Handbook</h6>
                                         <h6 class="text-primary mb-md-1">ScholarX</h6>
                                     </div>
                                 </a>
-                                <a href="/engineering-team.html" class="media d-flex align-items-center">
+                                <a href="../../../engineering-team/index.html" class="media d-flex align-items-center">
```

Comment:
Again we don't need this `index.html` part no? @RandilaP @anjula-sack 
```suggestion
                                <a href="../../../engineering-team/" class="media d-flex align-items-center">
```